android/src/ch/opengis/qfield/QFieldCameraPictureActivity.java

```diff
@@ -68,7 +69,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         Log.d(TAG, "resultCode: "+resultCode);
 
         File result = new File(prefix+pictureFilePath);
-        result.getParentFile().mkdirs();
+        File path = result.getParentFile();
+        path.mkdirs();
+
+        // Let the android scan new media folers/files to make them visible through MTP
+        path.setExecutable(true);
```

Comment:
Why the directory should be executable?

Reply:
The code is mostly taken from https://stackoverflow.com/a/30697198/2319028
I expanded from linux file permissions (e.g. https://askubuntu.com/a/862293/452931 "Directory execute bit - You need this permission if you want to: Have any access to files within the directory.")

I'm not sure we need to set permissions at all, this is just based on the above stackoverflow link.